db/migrate/20190901202623_create_user_locations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUserLocations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :user_locations do |t|
+      t.integer :user_id
```

Comment:
Could you change `integer` to `references` ? 
https://stackoverflow.com/questions/4736879/rails-is-there-a-difference-between-references-foo-and-integer-foo-id

Reply:
Done.

Reply:
And it's added indexes! great :)